File: app/src/main/java/com/ipd/taxiu/ui/activity/balance/MyBalanceActivity.java
package com.ipd.taxiu.ui.activity.balance;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import com.ipd.taxiu.R;
import com.ipd.taxiu.ui.BaseUIActivity;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * Created by Miss on 2018/7/30
 * 我的余额
 */
public class MyBalanceActivity extends BaseUIActivity implements View.OnClickListener{
    private TextView btn_account_balance,btn_withdraw_deposit,btn_my_bank_card;
    @Override
    protected int getContentLayout() {
        return R.layout.activity_my_balance;
    }

    @Override
    protected void initView(@Nullable Bundle bundle) {
        initToolbar();
        btn_account_balance = findViewById(R.id.btn_account_balance);
        btn_withdraw_deposit = findViewById(R.id.btn_withdraw_deposit);
        btn_my_bank_card = findViewById(R.id.btn_my_bank_card);
    }

    @Override
    protected void loadData() {

    }

    @Override
    protected void initListener() {
        btn_account_balance.setOnClickListener(this);
        btn_withdraw_deposit.setOnClickListener(this);
        btn_my_bank_card.setOnClickListener(this);
    }

    @NotNull
    @Override
    protected String getToolbarTitle() {
        return "我的余额";
    }

    @Override
    public void onClick(View v) {
        Intent intent;
        switch (v.getId()){
            case R.id.btn_account_balance:
                intent = new Intent(this,BalanceBillActivity.class);
                startActivity(intent);
                break;
            case R.id.btn_withdraw_deposit:
                intent = new Intent(this,WithdrawDepositActivity.class);
                startActivity(intent);
                break;
            case R.id.btn_my_bank_card:
                intent = new Intent(this,BankCardActivity.class);
                intent.putExtra("bankType","添加银行卡");
                startActivity(intent);
                break;
        }
    }
}

File: app/src/main/java/com/ipd/taxiu/ui/activity/balance/MyBalanceActivity.kt
package com.ipd.taxiu.ui.activity.balance

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.ipd.taxiu.R
import com.ipd.taxiu.ui.BaseUIActivity
import kotlinx.android.synthetic.main.activity_my_balance.*

/**
Created by Miss on 2018/8/10
我的余额
 */
class MyBalanceActivity : BaseUIActivity() {
    //1.添加银行卡 2.修改银行卡 3.选择银行卡
    companion object {
        val ADD_BANK_CARD = 1
        val UPDATE_BANK_CARD = 2
        val CHOSSE_BANK_CARD = 3

        fun launch(activity: Activity) {
            val intent = Intent(activity, MyBalanceActivity::class.java)
            activity.startActivity(intent)
        }
    }

    override fun getContentLayout(): Int = R.layout.activity_my_balance
    override fun getToolbarTitle(): String = "我的余额"

    override fun initView(bundle: Bundle?) {
        initToolbar()
    }

    override fun loadData() {
    }

    override fun initListener() {
        btn_account_balance.setOnClickListener { BalanceBillActivity.launch(this) }
        btn_withdraw_deposit.setOnClickListener { WithdrawDepositActivity.launch(this) }
        btn_my_bank_card.setOnClickListener { BankCardActivity.launch(this, ADD_BANK_CARD) }
    }
}